plugin/pom.xml

```diff
@@ -231,6 +232,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>io.jenkins.plugins</groupId>
+      <artifactId>pull-request-monitoring</artifactId>
+      <version>${pull-request-monitoring.version}</version>
+    </dependency>
```

Comment:
Could this dependency be optional? The more plugins server administrators have to install, the higher the risk that one of them has a vulnerability.

Reply:
Yes,  I think this can be marked as optional